docs_src/use-cases/automated-self-checkout/advanced.md

```diff
@@ -33,18 +33,22 @@ Environment variable with docker compose up
 
 ### Editing the Environment Files
 
-Environment variable files can be used to persist environment variables between deployments. There are three environment variables files with our default environment variables for Automated Self Checkout.
+Environment variable files can be used to persist environment variables between deployments. You can find these files in `src/res/` folder with our default environment variables for Automated Self Checkout.
 
-        - `src/gst.env` file for shared environment variables
-        - `src/yolov5-cpu.env` file for running pipeline on cpu only
-        - `src/yolov5-gpu.env` file for running pipeline in gpu or mixed CPU/GPU
+        - `src/res/all-cpu.env` file for running pipeline on CPU (decoding, pre-processing and inferencing)
+        - `src/res/all-gpu.env` file for running pipeline on GPU (decoding, pre-processing and inferencing)
+        - `src/res/all-dgpu.env` file for running pipeline on discrete GPU (decoding, pre-processing and inferencing)
+        - `src/res/all-npu.env` file for running pipeline on NPU (inferencing)
+        - `src/res/yolov5-cpu-class-gpu.env` file for running pipeline on CPU for detection and GPU for classification
+        - `src/res/yolov5-gpu-class-cpu.env` file for running pipeline on GPU for detection and CPU for classification
 
-After modifying or creating a new .env file you can load the .env file through docker compose up
+After modifying or creating a new .env file you can load the .env file through the make command or docker compose up
 
 !!! Example
 
     ```bash
-    docker compose -f src/docker-compose.yml --env-file src/res/yolov5-cpu.env up -d
+    make PIPELINE_SCRIPT=yolov5s_effnetb0.sh DEVICE_ENV=res/all-gpu.env run-render-mode
```

Comment:
Can we separate this into 2 examples - and say `!!! Example - Make` and `!!! Example - Docker Compose`? That way if you copy the command and run it you are not doing both at the same time?